ai_news_generator/app.py

```
@@ -1,165 +1,329 @@
 import os
+import sys
 import streamlit as st
-from crewai import Agent, Task, Crew, LLM
-from crewai_tools import SerperDevTool
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)